resources/options_values.md

```diff
@@ -186,78 +186,3 @@ Description: for inferring likely related isolates based on pairwise distance of
-
-#### Aditional configuration
-
- **--continue** If a module was chosen with the **--step** OPTION, the **--continue** OPTION ensures that the pipeline will continue the analysis with downstream modules. This is automatically set if the **--step** OPTION is set to the VALUE **TBfull**.
-
- **--samples** This OPTION requires a user supplied file specifying a set of datasets (e.g. samples.txt) as VALUE. The file must be a two-column, tab-separated file. Column 1 has to be your **[SampleID]**. Column 2 has to be your **[LibID]**. **TBjoin** requires this OPTION to be set.
-
- **--project** This OPTION allows the user to set a project name for the steps **TBjoin**, **TBamend** and **TBgroups**. If you do not support a project name, **[NONE]** is used as a default value.
-
- **--ref** This OPTION sets the reference genome for the read mapping. By default, the genome of <i>Mycobacterium tuberculosis</i> H37Rv (NC_000962.3) is set as reference. User supplied FASTA files for other reference genomes should be placed in the directory /MTBseq_source/var/ref/, and the respective name given without .fasta extension. Please be aware that for other reference genomes, you need to provide the respective annotation files as well or annotations will be skipped.
-
- **--resilist** This OPTION sets a list of known variant positions associated to drug resistance for resistance prediction. Give the full path to the file. The required structure of the file can be seen here: /MTBseq_source/var/res/MTB_Resistance_Mediating.txt
- 
- **--intregions** This OPTION sets a list of interesting regions to be used for annotation of detected variants. Give the full path to the file. The required structure of the file can be seen here: /MTBseq_source/var/res/MTB_Extended_Resistance_Mediating.txt
-
- **--categories** This OPTION specifies a gene categories file to annotate essential and non-essential genes as well as repetitive regions. SNPs in repetitive regions will be excluded for phylogenetic analysis. Give the full path to the file. The required structure of the file can be seen here: /MTBseq_source/var/cat/MTB_Gene_Categories.txt
-
- **--basecalib** This OPTION specifies a file for base quality recalibration. The list must be in VCF format and should contain known SNPs. Give the full path to the file. The required structure of the file can be seen here: /MTBseq_source/var/res/MTB_Base_Calibration_List.vcf
-
- **--all_vars** This OPTION is used in the modules **TBvariants**, **TBstats**, **TBjoin**, and **TBstrains**. By default, the OPTION is not active. Setting this OPTION will skip all filtering steps and report the calculated information for all positions in the input file.
-
- **--snp_vars** This OPTION is used in **TBvariants**, **TBstats**, **TBjoin**, and **TBstrains**. By default, the OPTION is not active. Setting this OPTION will add an additional filter that excludes all variants except SNPs.
```

Comment:
@Mxrcon , reading the codebase for the current PR, I'm not sure if we have made use of these features at all.

This means that for example, any user would only be able to run `MTBSeq` with the bundled reference fasta file since 


Similarly, the absence of `--basecalib` takes care of the base call calibration

>  The calibration list is stored in the directory "var/res/MTB_Base_Calibration_List.vcf" of the package. For other reference genomes, the file needs to be specified with the --basecalib OPTION or this step will be skipped.
                 
Basically, the actual design challenge of this workflow is not the DSL2 modules etc, but the treatment of parameters. This is where you need to use creativity and experience 😉

Reply:
I'm think that maybe we can tackle this in a follow up PR.

I've tracked this here 
https://github.com/emilyncosta/mtbseq-nf/issues/9

Reply:
yes, I think that for this, should create a specific PR